demo/manufacturer/README.md

```diff
@@ -74,6 +74,10 @@ Some required runtime arguments
 
   Default value: MfgApiPass123
 
+## Support for OnDie devices
+
+Refer to [Demo README](demo/README.md) for steps to configure manufacturer to support OnDie devices.
```

Comment:
The link should be (../README.md) in place of (demo/README.md). The current links for all 3 of manufacturer, owner and rv is resulting in 404 Not found error.

Reply:
oops, fixed the links